src/features/shared/hooks/use-live-price.tsx

```diff
@@ -0,0 +1,6 @@
+import { useLivePrices } from './use-live-prices';
```

Comment:
maybe move this to the use-live-prices file, just so that its easier to find all hooks related to live prices? just w suggestion